tests/plugins/clipboard.test.ts

```diff
@@ -190,16 +192,77 @@ describe("clipboard", () => {
     expect(getCell(model, "C2")!.style).toEqual({ bold: true });
   });
 
+  test("copying external content & paste-format on a cell will not paste content", async () => {
+    const model = new Model();
+    const clipboardData = new MockClipboardData();
+    clipboardData.setData(ClipboardMIMEType.PlainText, "Excalibur");
+
+    const content = clipboardData.getData(ClipboardMIMEType.PlainText);
+    pasteFromOSClipboard(model, "C2", content);
+    expect(getCellContent(model, "C2")).toBe(content);
+    pasteFromOSClipboard(model, "C3", content, "onlyFormat");
+    expect(getCellContent(model, "C3")).toBe("");
+  });
+
+  test("Internal copy followed by OS copy should not bring paste format from internal copy", () => {
+    const model = new Model();
+    setCellContent(model, "A1", "a1");
+    setStyle(model, "A1", { fillColor: "#FA0000" });
+    copy(model, "A1");
+    expect(getStyle(model, "A1")).toEqual({ fillColor: "#FA0000" });
+
+    pasteFromOSClipboard(model, "C4", "Excalibur", "onlyFormat");
+    expect(getCellContent(model, "C4")).toBe("");
+    expect(getStyle(model, "C4")).toEqual({});
+
+    paste(model, "C5", "onlyFormat");
+    expect(getCellContent(model, "C5")).toBe("");
+    expect(getStyle(model, "C5")).toEqual({});
+  });
+
+  test("cannot paste multiple times after cut", () => {
+    const model = new Model();
+    setCellContent(model, "B2", "b2");
+    expect(getCellContent(model, "B2")).toBe("b2");
+    setStyle(model, "B2", { bold: true });
+    expect(getCell(model, "B2")!.style).toEqual({ bold: true });
```

Comment:
```suggestion
    setCellContent(model, "B2", "b2");
    setStyle(model, "B2", { bold: true });
```
we can trust `setCellContent/Style` to do their jobs; we don't need to test every single thing. Same in the other tests.